src/tenderly/spawn-devnet.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-var-requires, no-console */
+import fs from "fs";
+import dotenv from "dotenv";
+import util from "util";
+import proc from "child_process";
+
+const asyncExec = util.promisify(proc.exec);
+
+dotenv.config();
+
+const {
+  TENDERLY_ACCESS_KEY,
+  TENDERLY_PROJECT_SLUG,
+  TENDERLY_ACCOUNT_ID,
+  TENDERLY_DEVNET_TEMPLATE,
+} = process.env;
+
+// eslint-disable-next-line max-len
+const command = `tenderly devnet spawn-rpc --project ${TENDERLY_PROJECT_SLUG} --template ${TENDERLY_DEVNET_TEMPLATE} --account ${TENDERLY_ACCOUNT_ID}  --access_key ${TENDERLY_ACCESS_KEY}`;
```

Comment:
You can use `\` to have multiline commands in code instead of disabling eslint

``